packages/curve-ui-kit/src/features/market-details/index.tsx

```diff
@@ -0,0 +1,181 @@
+import { formatNumber, FORMAT_OPTIONS } from '@ui/utils/utilsFormat'
+import { t } from '@ui-kit/lib/i18n'
+import { Metric } from '@ui-kit/shared/ui/Metric'
+import { SymbolCell } from '@ui-kit/shared/ui/SymbolCell'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { abbreviateNumber, scaleSuffix } from '@ui-kit/utils/number'
+
+const { Spacing } = SizesAndSpaces
+
+type Collateral = {
+  total: number | undefined | null
+  totalUsdValue: number | undefined | null
+  symbol: string | undefined | null
+  tokenAddress: string | undefined | null
+  usdRate: number | undefined | null
+  loading: boolean
+}
+type BorrowToken = {
+  total?: number | undefined | null
+  totalUsdValue?: number | undefined | null
+  symbol: string | undefined | null
+  tokenAddress: string | undefined | null
+  usdRate: number | undefined | null
+  loading: boolean
+}
+type BorrowAPY = {
+  value: number | undefined | null
+  thirtyDayAvgRate: number | undefined | null
+  loading: boolean
+}
+type LendingAPY = {
+  value: number | undefined | null
+  thirtyDayAvgRate: number | undefined | null
+  loading: boolean
+}
+type AvailableLiquidity = {
+  value: number | undefined | null
+  max: number | undefined | null
+  loading: boolean
+}
+type MaxLeverage = {
+  value: number | undefined | null
+  loading: boolean
+}
+
+export type MarketDetailsProps = {
+  collateral: Collateral
+  borrowToken: BorrowToken
+  borrowAPY: BorrowAPY
+  lendingAPY?: LendingAPY
+  availableLiquidity: AvailableLiquidity
+  maxLeverage?: MaxLeverage
+}
+
+const formatLiquidity = (value: number) =>
+  `${formatNumber(abbreviateNumber(value), { ...FORMAT_OPTIONS.USD })}${scaleSuffix(value).toUpperCase()}`
+
+export const MarketDetails = ({
+  collateral,
+  borrowToken,
+  borrowAPY,
+  lendingAPY,
+  availableLiquidity,
+  maxLeverage,
+}: MarketDetailsProps) => {
+  const utilization =
+    availableLiquidity?.value && availableLiquidity.max
+      ? (availableLiquidity.value / availableLiquidity.max) * 100
+      : undefined
+  const utilizationBreakdown =
+    availableLiquidity?.value && availableLiquidity.max
+      ? `${formatLiquidity(availableLiquidity.value)}/${formatLiquidity(availableLiquidity.max)}`
+      : undefined
+
+  return (
+    <Box sx={{ backgroundColor: (t) => t.design.Layer[1].Fill }}>
+      <CardHeader title={t`Market Details`} size={'small'} />
+      <Box
+        display="grid"
+        gap={3}
+        sx={{
+          padding: Spacing.md,
+          gridTemplateColumns: '1fr 1fr',
+          // 550px
+          '@media (min-width: 33.125rem)': {
+            gridTemplateColumns: '1fr 1fr 1fr 1fr',
+          },
+        }}
+      >
+        <Metric
+          size={'medium'}
+          label={t`Borrow rate`}
+          value={borrowAPY?.value}
+          loading={borrowAPY?.value == null && borrowAPY?.loading}
+          valueOptions={{ unit: 'percentage' }}
+          notional={
+            borrowAPY?.thirtyDayAvgRate
```

Comment:
Not just here, but in a lot of `<Metric />` usages in this PR you use a ternary in notionals. I think you can get away using the `&&` operator instead?

 ```ts
 borrowAPY?.thirtyDayAvgRate && { value: borrowAPY.thirtyDayAvgRate, unit: { symbol: '% 30D Avg', position: 'suffix' }}
```